rclpy/src/rclpy/_rclpy.c

```diff
@@ -501,6 +501,12 @@ rclpy_init(PyObject * Py_UNUSED(self), PyObject * args)
       }
       // Borrows a pointer, do not free arg_values[i]
       arg_values[i] = PyUnicode_AsUTF8(pyarg);
+      if (NULL == arg_values[i]) {
+        PyErr_Format(
```

Comment:
`PyUnicode_AsUTF8` already sets an exception on error. I think you can omit this statement.

Reply:
True that. Fixed!